tools/clean-tutorial-base.sh

```diff
@@ -7,5 +7,5 @@ set -e -u
 clean_tutorial .
 clean_aste 
 clean_precice_logs .
-rm -fv *.log
+rm -fv -- *.log
```

Comment:
For consistency with other places where we do a similar operation, I would rather do:

```suggestion
rm -fv ./*.log
```

Interestingly, this is not being triggered on my system, with shellcheck 0.8.0.